geyser-plugin-interface/src/geyser_plugin_interface.rs

```diff
@@ -62,12 +62,32 @@ pub struct ReplicaTransactionInfo<'a> {
     pub transaction_status_meta: &'a TransactionStatusMeta,
 }
 
+/// Information about a transaction, including index in block
+#[derive(Clone, Debug)]
+pub struct ReplicaTransactionInfoWithIndex<'a> {
+    /// The transaction's index in the block
+    pub index: usize,
```

Comment:
Does it make sense to put this new field at the end of the structure?

Reply:
You tell me, @lijunwangs . Is there a reason from the receiving side to add it to the end?

Reply:
The tangental benefit is probably for the code which did not recompile with new interface, and running against the updated validator, the code will more likely to work with  the new field in putting the back.

Reply:
Fair, okay will change